it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -190,229 +190,229 @@ sendAnyway=Send anyway
-subjectDlogMessage=You did not specify a subject for this message. If you would like to provide one, please type it now.
+subjectDlogMessage=Non hai specificato un oggetto per questo messaggio. Se vuoi inserirne uno, digitalo ora.
 
 ## String used by the dialog that informs the user about the newsgroup recipient
-recipientDlogMessage=Questo account gestisce solo destinatari di posta. Se si continua saranno ignorati i gruppi di discussione.
+recipientDlogMessage=Questo account supporta solo destinatari e-mail. Se continui i newsgroup saranno ignorati.
 
 #String used by the alert that tells the user an e-mail address is invalid
-addressInvalid=%1$S non è un indirizzo e-mail valido in quanto non è nella forma utente@server. Deve essere corretto prima di inviare l'e-mail.
+addressInvalid=%1$S non è un indirizzo e-mail valido poiché non è nella forma utente@server. Devi correggerlo prima di inviare l'e-mail.
 ## String used by the dialog that ask the user to attach a web page
-attachPageDlogTitle=Indicare l'indirizzo da allegare
+attachPageDlogTitle=Indica un percorso da allegare
 attachPageDlogMessage=Pagina web (URL):
 
 ## String used for attachment pretty name, when attachment is a message
 messageAttachmentSafeName=Messaggio allegato
 ## String used for attachment pretty name, when attachment is message part
-partAttachmentSafeName=Parte allegato al messaggio
+partAttachmentSafeName=Allegato parte del messaggio
 
 ## String used by the Initialization Error dialog
 initErrorDlogTitle=Composizione messaggio
 ## LOCALIZATION NOTE (initErrorDlogMessage): %1$S will be replaced by the error string.  do not translate \n
-initErrorDlogMessage=Si è verificato un errore durante la creazione di una finestra per la composizione del messaggio. Riprovare nuovamente. %1$s
+initErrorDlogMessage=Si è verificato un errore durante la creazione di una finestra per la composizione del messaggio. Si prega di riprovare di nuovo. %1$s
 
 ## Strings used by Save as Draft/Template dialog
-SaveDialogTitle=Salvataggio messaggio
+SaveDialogTitle=Salva messaggio
 ## LOCALIZATION NOTE (SaveDialogMsg): %1$S is folder name, %2$S is host name
-SaveDialogMsg=Il messaggio è stato salvato nella cartella %1$S in %2$S.
-CheckMsg=Non mostrare questa finestra in futuro.
+SaveDialogMsg=Il tuo messaggio è stato salvato nella cartella %1$S sotto %2$S.
+CheckMsg=Non mostrare questa finestra di nuovo.
 
 ## Strings used by prompt when Quitting while in progress
-quitComposeWindowTitle=Invio messaggio
+quitComposeWindowTitle=Invio messaggio in corso
 ## LOCALIZATION NOTE (quitComposeWindowMessage): don't translate \n
-quitComposeWindowMessage=Postbox is currently in the process of sending or saving a message.\nWould you like to wait until the message has been sent or saved before quitting or quit now?
+quitComposeWindowMessage=Postbox sta attualmente inviando o salvando un messaggio.\nVuoi attendere di inviare o salvare il messaggio prima di chiudere o vuoi chiudere ora?
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+quitComposeWindowMessage=Postbox sta attualmente inviando o salvando un messaggio.\nVuoi aspettare fino a quando il messaggio sarà stato inviato o salvato prima di uscire o vuoi uscire ora?